portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,49 +31,81 @@
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+
+@WebServlet("/new-post")
 public class DataServlet extends HttpServlet {
  private ArrayList<String> check; 
  public void init() {
+
     check = new ArrayList<String>();
```

Comment:
you can just initialize this as "private List<String> check = new ArrayList<>();"